app/Helpers/Vatsim.php

```diff
@@ -6,6 +6,18 @@
 
 class Vatsim
 {
+    private static $divisions = [
+        'CAN' => ['ZQM', 'ZUL', 'ZVR', 'ZWG', 'ZYZ'],
+        'CAR' => ['KIN', 'NAS', 'PAP', 'PIA', 'SDO', 'SJU'],
+        'EUD' => ['ADRIA', 'AUST', 'BLUX', 'BULG', 'CYPR', 'CZCH', 'ESTN', 'FINL', 'FRA', 'GEO', 'GER', 'GRE', 'HUN', 'ICL', 'IRL', 'ITA', 'LATVIA', 'LITH', 'MALT', 'MLDV', 'NETH', 'POL', 'POR', 'ROM', 'SCA', 'SLO', 'SPN', 'SUI', 'TURK', 'UKRA'],
+        'MENA' => ['ARB', 'DZA', 'EGYPT', 'IRN', 'JOR', 'KWIQ', 'KWT', 'LBY', 'LEB', 'MAG', 'MAR', 'NEA', 'SAU', 'SYR', 'YEM'],
+        'PAC' => ['PYF'],
+        'SAM' => ['ARG', 'BOL', 'CHI', 'COL', 'ECU', 'GUY', 'PAR', 'PER', 'URU', 'VEN'],
+        'SEA' => ['HK', 'IDN', 'MYR', 'MYS', 'PHL', 'SING', 'THA', 'VCL'],
+        'USA' => ['PCF', 'ZAB', 'ZAU', 'ZBW', 'ZDC', 'ZDV', 'ZFW', 'ZHU', 'ZID', 'ZJX', 'ZKC', 'ZLA', 'ZLC', 'ZMA', 'ZME', 'ZMP', 'ZNY', 'ZOA', 'ZOB', 'ZSE', 'ZTL'],
+        'WA' => ['AFG', 'BGD', 'BHU', 'IND', 'MDV', 'NPL', 'PAK', 'SLA', 'SRM'],
+    ];
```

Comment:
thought: should perhaps be available or specified in the configuration rather than in the helper directly?

Reply:
Which config do you have in mind? 
This will however not be needed if we just ask in .env for DIVISION_API_DRIVER="VATEUD"
(that's maybe what you meant?)